src/AppBundle/Entity/User.php

```diff
@@ -232,6 +232,26 @@ class User implements AdvancedUserInterface
     private $adManaged;
 
     /**
+     * @JMS\Type("string")
+     * @JMS\Groups({"user_details_pa"})
+     * @Assert\NotBlank( message="user.jobTitle.notBlank", groups={"user_details_pa"} )
+     * @Assert\Length(max=150, maxMessage="user.jobTitle.maxMessage", groups={"user_details_pa"} )
+     *
+     * @var string
+     */
+    private $jobTitle;
+
+    /**
+     * @JMS\Type("string")
+     * @JMS\Groups({"user_details_pa"})
+     * @Assert\NotBlank( message="user.paTeamName.notBlank", groups={"user_details_pa"} )
+     * @Assert\Length(max=50, maxMessage="user.paTeamName.maxMessage", groups={"user_details_pa"} )
+     *
+     * @var string
+     */
+    private $paTeamName;
```

Comment:
I've removed it from the API User.paTeamName, but this entity is associated to the Form to save/load the team name. We'll rediscuss in the next PR where I'll show the implementation (with logic to store/load team name in the API, and not in the client)